src/common/input/keybinds.rs

```diff
@@ -29,7 +29,7 @@ pub struct KeybindsFromYaml {
 #[serde(untagged)]
 enum KeyActionUnbind {
     KeyAction(KeyActionFromYaml),
-    Unbind(UnbindFromYaml),
```

Comment:
This breaks current configurations.
You are not able to anymore unbind keys in this way:

```
keybinds:
    normal:
        - unbind : true
        - action: [GoToTab: 1,]
          key: [F: 1,]
```

or 

```
keybinds:
    normal:
        - unbind : [Alt: 'n', Ctrl: 'p']
        - action: [GoToTab: 1,]
          key: [F: 1,]
```

Reply:
I think I might need your help with understanding the intended functionality here. I removed it on the basis of it looking potentially redundant (since `struct UnbindFromYaml` just contained `enum Unbind`, which I put in as its replacement).

However, when I set the default config to contain both of your examples:
- All the tests pass
- I'm not sure anything is actually being unbound

Both with and without my change.

Reply:
`example/default.yaml` is not `assets/config/default.yaml` 😄

Reply:
The intended functionality there is that you can be granular in the unbinding configuration, since we add a lot of default keybindings and a person might just want to remove all the alt keybindings, or maybe just all the alt keybindings for a specific mode. Or wants to just unbind a specific mode.

https://zellij.dev/documentation/keybindings.html#to-unbind-the-default-keybindings

Reply:
Cool - that makes a lot of sense as a goal for what the functionality should be.

I just hadn't noticed that the `default.yaml` I was editing was the one from the example folder

Reply:
Ah, no problem! Yes that can be confusing and I think at some point we should remove it. I think though atm it is still a good reference.

Yes, the struct looks redundant. It is a way to tell serde that we want the unbind keyword there and match an untagged enum, if we don't have a struct there we probably would need to write our own deserialiser implementation, at least if I understood everything correctly.